scripts/variantstore/wdl/CreateCohortTable.wdl

```diff
@@ -0,0 +1,88 @@
+version 1.0
+
+workflow CreateCohortTable {
+   input {
+        String data_project
+        String default_dataset
+        String destination_cohort_table_name
+
+        # inputs with defaults
+        String query_project = data_project
+        String destination_project = data_project
+        String destination_dataset = default_dataset
+
+        String fq_petvet_dataset = "~{data_project}.~{default_dataset}"
+        String fq_cohort_sample_table = "~{data_project}.~{default_dataset}.sample_info"
+        String fq_sample_mapping_table = "~{data_project}.~{default_dataset}.sample_info"
+        String fq_temp_table_dataset = "~{destination_project}.temp_tables"
+        String fq_destination_dataset = "~{destination_project}.~{destination_dataset}"
+
+        String? docker
+    }
+
+    # TODO update this docker source
+    String docker_final = select_first([docker, "us.gcr.io/broad-dsde-methods/variantstore:latest"])
```

Comment:
the `latest` tag doesn't automatically work - need help figuring out what to do here